src/app/modules/account/pages/spouse-info/remove-spouse/remove-spouse.component.html

```diff
@@ -59,25 +59,35 @@ <h2>
 </div>
 
 <div *ngIf="spouse.cancellationReason === 0">
-  <common-radio
-    [value]="spouse.hasCurrentMailingAddress"
-    label="Do you know your spouse's current mailing address?"
-    [radioLabels]='[{"label": "No", "value": false}, {"label": "Yes", "value": true}]'
-    (valueChange)="spouse.hasCurrentMailingAddress = $event; this.dataService.saveMspAccountApp();">
-  </common-radio>
-
-  <div *ngIf="spouse.hasCurrentMailingAddress">
-    <common-page-section
-      layout='double'>
+  <div class="row">
+    <div class="col-sm-12 col-md-4 form-group">
+      <common-radio
+        [value]="spouse.hasCurrentMailingAddress"
+        label="Do you know your spouse's current mailing address?"
+        [radioLabels]='[{"label": "No", "value": false}, {"label": "Yes", "value": true}]'
+        (valueChange)="spouse.hasCurrentMailingAddress = $event; this.dataService.saveMspAccountApp();">
+      </common-radio>
+    </div>
+  </div>
+  <common-page-section layout="noTips">
+    <div *ngIf="spouse.hasCurrentMailingAddress">
       <h2><strong>Spouse's Mailing Address</strong></h2>
       <p class="horizontal-line">Please provide the spouse's current mailing address.</p>
-      <common-address
-        [address]="spouse.mailingAddress"
-        (addressChange)="handleAddressUpdate($event)"
-        [isRequired]="true">
-      </common-address>
-    </common-page-section>
-  </div>
+      <div class="row">
+        <div class="col-sm-12 col-md-4 form-group">
+          <common-address
+            #address
+            [address]="spouse.mailingAddress"
+            (addressChange)="handleAddressUpdate($event)"
+            [allowExtralines]="true"
```

Comment:
Question: Do all the address lines now get added to the JSON payload?

Reply:
Not yet, but I'll add them in my next push.